t/10-tests_overview.t

```diff
@@ -23,6 +23,7 @@ sub get_summary { OpenQA::Test::Case::trim_whitespace($t->tx->res->dom->at('#sum
 
 my $jobs = $schema->resultset('Jobs');
 $jobs->find(99928)->update({blocked_by_id => 99927});
+$jobs->find($_)->comments->create({text => 'test1', user_id => 99901}) for 99946, 99963;
```

Comment:
Maybe don't make it an exact match for the search string? So you can be sure substring match works.